pkg/filestore/filestore.go

```diff
@@ -24,8 +24,10 @@ import (
 	"github.com/tus/tusd/v2/pkg/handler"
 )
 
-var defaultFilePerm = os.FileMode(0664)
-var defaultDirectoryPerm = os.FileMode(0754)
+var (
```

Comment:
Can you please remove these unrelated changes?

Reply:
Ahh - autoformatter stuff.